generate-requirements-development-lock.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+virtualenv /tmp/generate-requirements-development-lock
+source /tmp/generate-requirements-development-lock/bin/activate
+pip install -r ${BASH_SOURCE%/*}/concent_api/requirements.lock
```

Comment:
We settled with @cameel  to use `python -m pip install -r "requirements.lock"` form. What do you think about this @cameel ?

Reply:
It's a bit safer but it does not matter all that much here. This script is only used in development.

The difference between `pip` and `python -m pip` is only relevant when you have an old version of `pip` installed on the machine. That might be a case in a docker container but it's not all that likely on developer's machine.